zerver/lib/hotspots.py

```diff
@@ -86,6 +86,9 @@ def to_dict(self) -> Dict[str, str]:
     OneTimeNotice(
         name="inbox_modal",
     ),
+    OneTimeNotice(
+        name="recent_conversations_modal",
```

Comment:
I was willing to name these `intro_recent_conversations_modal` & `intro_inbox_modal`. But the character limit is 30 characters in our model. Are the current names okaish ?

Reply:
If it's annoying, we can just do a quick migration to remove or increase the character limit to 50? There's no good reason to have such a short character limit.

Reply:
But I think maybe `intro_recent_view_modal` and `intro_inbox_view_modal` might be better names that fit into the character limit anyway?